content/docs/extensions/deployment/sagemaker.md

```diff
@@ -4,7 +4,114 @@ Implements MlemEnv, MlemDeployment and DeployState to work with AWS SageMaker
 
 ## Description
 
-**TODO**
+MLEM SageMaker allow you to deploy MLEM models to AWS SageMaker. You can learn
+more about SageMaker
+[here](https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html).
```

Comment:
This should be the intro sentence TBH (before **Description** or maybe we don't even need a `## Description` header (maybe thus move this one change to #179?).

We can condense it even more too:

```suggestion
This extension allows you to deploy MLEM models to AWS [SageMaker].

[sagemaker]: https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html
```

One question on terminology though: is "MLEM model" the same as "MLEM model object"? Also sounds a lot like "ML model". We have to think this through so it's clear, I think. Maybe phrase it as `deploy Ml models from MLEM to `